src/login/login.component.css

```diff
@@ -39,7 +39,7 @@
 @media only screen and (min-width: 481px) {
   .login-card {
     padding: 3rem;
-    width: 20.375rem;
+    width: 26.375rem;
```

Comment:
Since `box-sizing` for all elements is set to `border-box`, I added 6 rem to the width to cater for the left and right padding